### PR TITLE
fix OS dependency for rstudio for Debian-based systems

### PR DESCRIPTION
**Pull request by crubb**
(created using `eb --new-pr`)
Add libpam0g-dev to osdependencies

### CODE REVIEW

**Comment by crubb** [comment]
Test report by @crubb
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
radbert - Linux ubuntu 18.04, Intel(R) Xeon(R) CPU E5-2687W v4 @ 3.00GHz, Python 2.7.17
See https://gist.github.com/64687699dc6072bcde35f07687a1bbcf for a full test report.

File: easybuild/easyconfigs/r/rstudio/rstudio-1.2.5033-fosscuda-2019b-Java-11.eb [hunk 1]
```diff
@@ -38,7 +38,9 @@ dependencies = [
     ('Java', '11', '', True),
 ]
 
-osdependencies = ['pam-devel']
+osdependencies = [
+    ('pam-devel', 'libpam0g-dev')
```
Comment (boegel) [inline easybuild/easyconfigs/r/rstudio/rstudio-1.2.5033-fosscuda-2019b-Java-11.eb:42]: @crubb Is it worth adding an additional constant for this in framework, like we did for OpenSSL packages in https://github.com/easybuilders/easybuild-framework/pull/3309 ?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, x86_64, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/caa7596c3eee987ac337e92a8e752109 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3301.joltik.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6242 CPU @ 2.80GHz (cascadelake), Python 3.6.8
See https://gist.github.com/c505d651397f86188876807430765101 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @crubb!

**Pull request merged**